docs/velocity/admin/reference/comparison.md

```diff
@@ -8,6 +8,16 @@ Presumably, you have learned about what Velocity can do for you. But how does it
 other solutions out there? We're trying to convince you to use Velocity, after all, so this document
 may be somewhat biased in our favor.
 
+## Overview
+| Feature                           	   | BungeeCord | Waterfall | Velocity |
+|------------------------------------------|------------|-----------|----------|
+| BungeeCord plugins                       | ✅          | ✅         | ❌        |
```

Comment:
It is somewhat possible to run Bungee plugins on Velocity using a plugin, not sure if that's reliable enough to mention that on the docs though